front/src/components/ArticleCreate.jsx

```diff
@@ -55,12 +55,12 @@ export default function ArticleCreate ({ onSubmit }) {
       delete createdArticle.addTags
       onSubmit(createdArticle)
       setToast({
-        text: `New article created`,
+        text: t('articleCreate.notification.newArticle'),
```

Comment:
Je suis un peu partagé sur le `notification`. Peut être quelque chose comme:

```suggestion
        text: t('article.create.successNotification'),
```